### PR TITLE
Completed centrifugo implementation

### PR DESCRIPTION
**Pull request by NdubuisiJr**
This pull request address the websocket issue for issue #232,  #228, and #227 and creates the infrastructure needed to implement other features

### CODE REVIEW

File: server/src/controllers/game.controller.js [hunk 1]
```diff
@@ -1,32 +1,105 @@
...
+const { save, retrieve, deleteData } = require("../utils/cacheData");
 
 // Custom Modules
 const response = require("../utils/response");
 const CustomError = require("../utils/custom-error");
+const centrifugoController = require("../controllers/centrifugoController");
 
 class GameController {
-  async create(req, res) {
+  create(req, res) {
```
Comment (eni4sure) [inline server/src/controllers/game.controller.js:11]: you removed async here?
Any reason?

Comment (NdubuisiJr) [reply server/src/controllers/game.controller.js:11]: There's nothing asynchronous in the function

**Comment by NdubuisiJr** [comment]
I'm merging this to dev for the sake of testing out everything live.
If it breaks something I'll revert the merge @eni4sure @Alao-Abiodun

**Comment by eni4sure** [comment]
alright boss!
I'll test now

**Pull request merged**